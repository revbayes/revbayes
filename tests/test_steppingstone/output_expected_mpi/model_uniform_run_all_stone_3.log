Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21359.97	-21337.3	-22.66301	0.8149889	0.161348	0.147402	0.181602	0.0350935	0.105404	0.240456	0.351979	0.149376	0.240581	0.107352	0.27218	0.224691	0.145859	0.262614	0.211638	0.190884	0.108884	0.17658	0.223837	0.254051	0.25345	0.151532	0.590415	0.137403	0.0375459	0.00294787	0.165279	0.0350047	0.0241987	0.128246	0.388418	0.0851868	0.130792	0.00104409	0.119679	0.0770075	0.0547534	0.067435	0.032515	0.0128058	0.159681	0.132278	0.327332	0.0290403	0.269242	0.0595962	0.0244405	0.570942	0.0467381	0.098744	0.413099	0.961023	2.52713	0.356375	0.325948	0.0997072	0.21797	0.3499651	6.906761
10	-21396.13	-21377.49	-18.64151	0.8965102	0.171368	0.15262	0.122081	0.152041	0.117725	0.186112	0.336181	0.0868422	0.29619	0.0878981	0.214908	0.156454	0.113638	0.15752	0.165648	0.0982424	0.0825922	0.24891	0.15895	0.275137	0.409942	0.183436	0.486835	0.0769544	0.053109	0.000926213	0.0347384	0.0545653	0.0857287	0.0796311	0.273543	0.0660213	0.0534845	0.00304299	0.151145	0.0708711	0.0431573	0.0804138	0.0350744	0.00937963	0.248178	0.173159	0.464169	0.0361266	0.338447	0.0542534	0.007821	0.506915	0.0564367	0.115871	0.443272	0.980486	2.46037	0.362727	0.352219	0.0780226	0.207032	0.3811242	6.518564
