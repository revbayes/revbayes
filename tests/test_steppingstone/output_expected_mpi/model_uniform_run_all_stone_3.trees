Iteration	Posterior	Likelihood	Prior	psi
0	-21359.97	-21337.3	-22.66301	(Daubentonia_madagascariensis[&index=7]:0.351979,((((Loris_tardigradus[&index=12]:0.224691,Nycticebus_coucang[&index=15]:0.211638)[&index=37]:0.054753,(Perodicticus_potto[&index=18]:0.176580,(Otolemur_crassicaudatus[&index=16]:0.190884,Galago_senegalensis[&index=8]:0.149376)[&index=24]:0.137403)[&index=29]:0.024199)[&index=41]:0.159681,(Propithecus_coquereli[&index=19]:0.223837,(((Microcebus_murinus[&index=14]:0.262614,Cheirogaleus_major[&index=4]:0.035094)[&index=35]:0.119679,(Lemur_catta[&index=10]:0.107352,Varecia_variegata_variegata[&index=22]:0.151532)[&index=38]:0.067435)[&index=40]:0.012806,Lepilemur_hubbardorum[&index=11]:0.272180)[&index=39]:0.032515)[&index=27]:0.165279)[&index=28]:0.035005,(Tarsius_syrichta[&index=21]:0.253450,(((Pan_paniscus[&index=17]:0.108884,Hylobates_lar[&index=9]:0.240581)[&index=32]:0.085187,(Colobus_guereza[&index=6]:0.240456,(Macaca_mulatta[&index=13]:0.145859,Chlorocebus_aethiops[&index=5]:0.105404)[&index=25]:0.037546)[&index=42]:0.132278)[&index=30]:0.128246,(Callicebus_donacophilus[&index=2]:0.147402,(Aotus_trivirgatus[&index=1]:0.161348,(Saimiri_sciureus[&index=20]:0.254051,Cebus_albifrons[&index=3]:0.181602)[&index=26]:0.002948)[&index=33]:0.130792)[&index=43]:0.327332)[&index=31]:0.388418)[&index=36]:0.077008)[&index=34]:0.001044,Galeopterus_variegatus[&index=23]:0.590415)[&index=44];
10	-21396.13	-21377.49	-18.64151	(Daubentonia_madagascariensis[&index=7]:0.336181,((Tarsius_syrichta[&index=21]:0.409942,(((Loris_tardigradus[&index=12]:0.156454,Nycticebus_coucang[&index=15]:0.165648)[&index=37]:0.043157,((Otolemur_crassicaudatus[&index=16]:0.098242,Galago_senegalensis[&index=8]:0.086842)[&index=24]:0.076954,Perodicticus_potto[&index=18]:0.248910)[&index=29]:0.085729)[&index=41]:0.248178,(Propithecus_coquereli[&index=19]:0.158950,((Lepilemur_hubbardorum[&index=11]:0.214908,(Cheirogaleus_major[&index=4]:0.152041,Microcebus_murinus[&index=14]:0.157520)[&index=35]:0.151145)[&index=39]:0.035074,(Varecia_variegata_variegata[&index=22]:0.183436,Lemur_catta[&index=10]:0.087898)[&index=38]:0.080414)[&index=40]:0.009380)[&index=27]:0.034738)[&index=28]:0.054565)[&index=34]:0.003043,(((Aotus_trivirgatus[&index=1]:0.171368,(Cebus_albifrons[&index=3]:0.122081,Saimiri_sciureus[&index=20]:0.275137)[&index=26]:0.000926)[&index=33]:0.053484,Callicebus_donacophilus[&index=2]:0.152620)[&index=43]:0.464169,(((Chlorocebus_aethiops[&index=5]:0.117725,Macaca_mulatta[&index=13]:0.113638)[&index=25]:0.053109,Colobus_guereza[&index=6]:0.186112)[&index=42]:0.173159,(Pan_paniscus[&index=17]:0.082592,Hylobates_lar[&index=9]:0.296190)[&index=32]:0.066021)[&index=30]:0.079631)[&index=31]:0.273543)[&index=36]:0.070871,Galeopterus_variegatus[&index=23]:0.486835)[&index=44];
