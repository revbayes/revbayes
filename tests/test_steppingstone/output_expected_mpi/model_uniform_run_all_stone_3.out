state	power	likelihood
4	0.16807	-21323.3
5	0.16807	-21330.8
6	0.16807	-21354.1
7	0.16807	-21359.6
8	0.16807	-21353.7
9	0.16807	-21393.4
10	0.16807	-21377.5
